### PR TITLE
Temporary login register logout

### PR DESCRIPTION
**Pull request by ichala**
Add a temporary Login/logout features so we can proceed with next tasks

### CODE REVIEW

**Review by EidHachem — APPROVED** [review]
Approved ✅

**Review by d2ndjim — APPROVED** [review]
Hi @ichala ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Great Implementation✔️

File: src/redux/slices/auth.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const authSlice = createSlice({
     },
     Logout: () => {
       localStorage.removeItem('user');
+      return null;
```
Comment (aimalamiri) [inline src/redux/slices/auth.js:26]: Good Job! I have a clarification question in here, what's the reason of returning null? why do we need this?

**Review by aimalamiri — APPROVED** [review]
### **Hi 👋** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/redux/slices/auth.js (hunk 1 above)
Comment (ichala) [reply src/redux/slices/auth.js:26]: @aimalamiri the Logout dispatch removes the user from local storage, return null is to update the user state (redux) to null, so it auto reload all components, if I don't pull return null the local storage gonna be deleted , but the redux state gonna still have the user object
same as login we log in and add the user to the redux state, logout will remove it from the state

**Pull request merged**